packages/terraform-modules-public/testnet/main.tf

```diff
@@ -0,0 +1,240 @@
+  geth_exporter_docker_image_repository = var.geth_exporter_docker_image_repository
+  geth_exporter_docker_image_tag        = var.geth_exporter_docker_image_tag
+  geth_node_docker_image_repository     = var.geth_node_docker_image_repository
+  geth_node_docker_image_tag            = var.geth_node_docker_image_tag
+  geth_verbosity                        = var.geth_verbosity
+  in_memory_discovery_table             = var.in_memory_discovery_table
+  istanbul_request_timeout_ms           = var.istanbul_request_timeout_ms
+  network_id                            = var.network_id
+  network_name                          = var.network_name
+  tx_node_count                         = var.tx_node_count
+  validator_count                       = var.validator_count
+  reset_geth_data                       = var.reset_geth_data
+
+  proxy_name                  = var.proxy_name
+  proxy_private_keys          = var.proxy_private_keys
+  validator_account_addresses = var.validator_account_addresses
+  validator_account_passwords = var.validator_account_passwords
+  validator_private_keys      = var.validator_private_keys
+  static_nodes_base64         = base64encode(data.http.static-nodes.body)
+}
+
+module "validator" {
+  source = "./modules/validator"
+  # variables
+  block_time                            = var.block_time
+  celo_env                              = var.celo_env
+  instance_type                         = var.instance_types["validator"]
+  ethstats_host                         = var.ethstats_host
+  genesis_content_base64                = base64encode(data.http.genesis.body)
+  geth_exporter_docker_image_repository = var.geth_exporter_docker_image_repository
+  geth_exporter_docker_image_tag        = var.geth_exporter_docker_image_tag
+  geth_node_docker_image_repository     = var.geth_node_docker_image_repository
+  geth_node_docker_image_tag            = var.geth_node_docker_image_tag
+  geth_verbosity                        = var.geth_verbosity
+  in_memory_discovery_table             = var.in_memory_discovery_table
+  istanbul_request_timeout_ms           = var.istanbul_request_timeout_ms
+  network_id                            = var.network_id
+  network_name                          = var.network_name
+  tx_node_count                         = var.tx_node_count
+  validator_count                       = var.validator_count
+  reset_geth_data                       = var.reset_geth_data
+
+  validator_name              = var.validator_name
+  validator_account_addresses = var.validator_account_addresses
+  validator_private_keys      = var.validator_private_keys
+  validator_account_passwords = var.validator_account_passwords
+  proxy_enodes                = var.proxy_enodes
+  proxy_internal_ips          = module.proxy.internal_ip_addresses
+  proxy_external_ips          = module.proxy.external_ip_addresses
+  static_nodes_base64         = base64encode(data.http.static-nodes.body)
```

Comment:
We can remove `static_nodes_base64` for the validator, right?

Reply:
Right thanks 👍